### PR TITLE
Project 2: Building with Responsive Design - Milestones 1

### PR DESCRIPTION
**Pull request by aliabbani**
## RESPONSIVE DESIGN
- Building with Responsive Design
- This collaborative project was built using HTML5 for the skeletal work
- CSS3 for styling
- Also the use of media queries to make it responsive
Note: Most of the reviews are being implemented except for the breakpoints. We started with the desktop version, and we created 2 breakpoints (480px for mobile & 768px for tablet).

### CODE REVIEW

**Review by Srjnnnn — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hello @aliabbani, @Whoistolu. Your page looks good on the desktop version. :muscle: However, before moving on there are some things that need to be done. :100: 

- [x]  I think it's better to make the necessary navigational links as links instead of giving the name and the links afterward. I mean, for example in your Readme your social links are not as links, so please work on it.
- [ ]  You've used two breakpoints which is good, but you should use 768px and 1024px, and all the default styles should be applied to the mobile version. So it would be good to change your media breakpoints to 768px and 1024px and please use a mobile-first approach.
- [x]  You need to use the nav tag instead of the div for your menu options. Kindly implement this.
- [x]  For the mobile version of the app, you need to have the logo leftmost side on the top. Kindly implement this.
- [x]  Next to the search icon, you need to have a hamburger bar for the mobile version. Please implement this.
- [x]  For the tablet version, the yellow part should have a top and bottom padding for the tablet version. Please implement this.
- [x]  In your footer the social media icons are placed into a nav tag and centered. Kindly implement this.

 Apart from those issues I mentioned, your page looks good and I highly believe you could easily solve those problems. :100: 
As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#) you have 2 code reviews left for this project.
_Eyüp Sercan UYGUR_

File: index.html [hunk 1]
```diff
@@ -43,8 +43,8 @@
                         <li>PLUGGED</li>
                         <li>README</li>
                         <li>GROWTH QUARTERS</li>
-                        <li>SHIFT</li>
-                        <li>NEURAL</li>
+                        <li id="dont">SHIFT</li>
+                        <li >NEURAL</li>
                     </ul>
                 </a>
             </div>
```
Comment (Srjnnnn) [inline index.html:62]: This container div should be a nav tag.

Comment (Srjnnnn) [reply index.html:62]: Same goes for the footer with your social icons.

**Review by Srjnnnn — CHANGES_REQUESTED** [review]

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @aliabbani and @Whoistolu :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by aliabbani** [comment]
merging milestone-1

**Pull request merged**